scripts/asm-installer/install_asm

```diff
@@ -212,9 +212,15 @@ main() {
         init_gcp_cas
       fi
       enable_stackdriver_kubernetes
+      if [[ "${USE_VM}" -eq 1 ]]; then
```

Comment:
Please create a `mesh_feature_required` function and add this check there instead of looking for VMs specifically.

Reply:
done